src/NUnitFramework/framework/Attributes/TestFixtureSourceAttribute.cs

```diff
@@ -33,7 +33,7 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// TestCaseSourceAttribute indicates the source to be used to
+    /// TestFixtureSourceAttribute indicates the source to be used to
```

Comment:
Hey, would you mind doing `<see cref=` while you're here? Thanks for doing this!